runtime/autoload/provider/clipboard.vim

```diff
@@ -172,6 +172,10 @@ function! s:clipboard.set(lines, regtype, reg) abort
   if jobid > 0
     call jobsend(jobid, a:lines)
     call jobclose(jobid, 'stdin')
+    if argv[0] ==# 'xclip'
```

Comment:
Tested this with by copying some text via Vim's clipboard first and then running `make oldtest`.